.eslintrc

```diff
@@ -479,7 +479,8 @@
     "error",
     "double",
     {
-      "avoidEscape": true
+      "avoidEscape": true,
+      "allowTemplateLiterals": true
```

Comment:
Why are we making this change? Generally we should never touch this

Reply:
Yes, when you encounter a discrepancy between linting and your code the path is to correct your code, not the lint rules. The lint rules are the same for the entire organization.